scipy/stats/_distribution_infrastructure.py

```diff
@@ -3651,6 +3669,49 @@ class or its methods for more information.
     >>> np.isclose(X.cdf(2.1), Y.cdf(2.1))
     np.True_
 
+    Create a custom distribution with multiple parameterizations. Here we create a
+    custom version of the Beta distribution that has an alternative parameterization
+    in terms of the mean ``mu`` and a dispersion parameter ``nu``.
+
+    >>> class MyBeta:
+    ...     @property
+    ...     def __make_distribution_version__(self):
+    ...         return "1.16.0"
+    ...
+    ...     @property
+    ...     def parameters(self):
+    ...         return (
+    ...             {"a": (0, np.inf), "b": (0, np.inf)},
+    ...             {"mu": (0, np.inf), "nu": (0, np.inf)},
```

Comment:
```suggestion
    ...             {"mu": (0, 1), "nu": (0, np.inf)},
```
Right?

Same in test.

Reply:
Yes. It popped in my mind that I did this wrong as I was falling asleep last night. It's because I just copy-pasted to correct the second entry, and only changed the parameter names.